nwutils/nw-builder
# feat(build): rename MacOS Helpers

Fixes: #966

## Description

- Rename MacOS Helper apps
- Update relevant Info.plist values
- Update Helper apps and relevant Info.plist values 
- Do not rely on symlinks

> Note: MacOS CI fails since #932 has not landed. Helper apps will also not be renamed correctly since symlinks are not preserved. 

## Testing steps

- Clone repo
- Checkout `macos-helpers` branch
- `npm install` node modules
- `npm run demo` to download and build demo app
- Go to `/test/fixture/out/Demo.app` and execute application
- Verify that app does not crash with error message `The display compositor is frequently crashing`
- Verify that Helper apps are renamed by checking in Activity Monitor
